### DISCUSSION TITLE
Custom theme  import bootstrap build error

### DISCUSSION THREAD

**Discussion by Genuifx** [vuepress/core#942] [Q&A] [answered]
`Layout.vue` import bootstrap js while `vuepress dev` works fine, but throw an error by `vuepress build`

![image](https://user-images.githubusercontent.com/10156994/173728432-a72de73f-b0c6-4c16-8b36-e2a77675905d.png)

vuepress version : vuepress@2.0.0-beta.46

**Comment by Mister-Hope** [comment, marked as answer]
https://v2.vuepress.vuejs.org/advanced/cookbook/usage-of-client-config.html#use-non-ssr-friendly-features
